ts/features/wallet/onboarding/paypal/screen/__tests__/PayPalPpsSelectionScreen.test.tsx

```diff
@@ -33,6 +41,16 @@ describe("PayPalPpsSelectionScreen", () => {
   it.todo(
     "error and retry button should be shown when some error occurred while retrieving data"
   );
+
+  it(`"what is a psp" link should be defined and open a bottom sheet on onPress`, () => {
+    const render = renderComponent(store);
+    const link = render.component.queryByTestId("whatIsPSPTestID");
```

Comment:
```suggestion
    const link = render.component.getByText(
      I18n.t("wallet.onboarding.paypal.selectPsp.link")
    );
    fireEvent.press(link);
    expect(mockPresentBottomSheet).toBeCalledTimes(1);
```

(opinionated 😸 ) Suggestions:
1. use `getBy*` when testing for the component's behavior rather than its presence, because its return type is not nullable and throws if not found.
2. look for actual text rather than IDs when possible. This will mimick how a real user finds an element on the UI.
3. use `queryBy*` only when testing for conditional presence on the UI (optional content)

Note for (2): https://testing-library.com/docs/dom-testing-library/intro#this-solution

Reply:
I'm on your side! thx